src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -1162,6 +1171,13 @@ void ObjectMgr::LoadCreatureAddons()
             creatureAddon.meleeAnimKit = 0;
         }
 
+        if (creatureAddon.visibilityDistanceType >= VisibilityDistanceType::Max)
+        {
+            TC_LOG_ERROR("sql.sql", "Creature (GUID: " UI64FMTD ") has invalid visibilityDistanceType (%u) defined in `creature_addon`.",
```

Comment:
%u is missing here too

Reply:
There is nothing missing here

Reply:
they do? https://github.com/TrinityCore/TrinityCore/blob/a78a85e8b8b5cf02df486c776fff578a153a9906/src/server/game/Globals/ObjectMgr.cpp#L1170